data.sql

```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
 ('Angemon','2005-06-12',-45.0,true,1),
 ('Boarmon','2005-06-07',20.4,true,7),
 ('Blossom','1998-10-13',17.0,true,3);
+
+INSERT INTO owners(full_name, age) 
+VALUES 
+('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES ('Pokemon'),('Digimon');
+
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = 4
+WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+
+UPDATE animals
+SET owner_id = 5
+WHERE name = 'Angemon' OR name = 'Boarmon';
+
+UPDATE animals
+SET species_id = 2
+WHERE name like '%mon';
+
+UPDATE animals
+SET species_id = 1 
+WHERE species_id IS NULL;
```

Comment:
- [ ] Let us modify our implementation here, you  are updating animals after  setting the owner_id to the value, imagine once you have many ids how would you handle it?

Hint: refer to following sample solution to answer the remaining:

`UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';`

Reply:
:+1: